testDownloadBehavior/README.md

```diff
@@ -0,0 +1,114 @@
+# WebdriverIO `Download Behavior` Examples
+
+This example directory contains test files that demonstrate WebdriverIO's download behavior.
+
+## Prerequisite
+
+Make sure to run this first, to set up the example repository:
+
+```sh
+npm install
+```
+
+## For Google Chrome
+
+**NOTE**: Ensure that the directory specified in `example.js` matches the directory you've set in capabilities.
+
+To test download behavior in **Google Chrome**, use these capabilties.
+
+```javascript
+{
+    browserName: "chrome",
+    "goog:chromeOptions": {
+        prefs: {
+            "download.default_directory": "/path/to/downloads/directory",
+        },
+    },
+}
+```
+
+## For Microsoft Edge
+
+**NOTE**: Ensure that the directory specified in `example.js` matches the directory you've set in capabilities.
+
+To test download behavior in **Microsoft Edge**, use these capabilties.
+
+```javascript
+{
+    browserName: "msedge",
+    "ms:edgeOptions": {
+        prefs: {
+            "download.default_directory": "/path/to/downloads/directory",
+        },
+    },
+}
+```
+
+## For Mozilla Firefox
+
+**NOTE**: Ensure that the directory specified in `example.js` matches the directory you've set in capabilities.
+
+To test download behavior in **Mozilla Firefox**, use these capabilties.
+
+```javascript
+{
+    browserName: "firefox",
+    "moz:debuggerAddress": true,
+    "moz:firefoxOptions": {
+        prefs: {
+            "browser.download.dir": "/path/to/downloads/directory",
+            "browser.download.folderList": 2,
+            "browser.download.manager.showWhenStarting": false,
+            "browser.helperApps.neverAsk.saveToDisk": "*/*",
+        },
+    },
+}
+```
+
+## For Apple Safari
+
+I have tried overriding default **Safari** download directory using:
+
+```javascript
+{
+    browserName: "safari",
+    "safari.safariOptions":{
+        dataDir: "/path/to/downloads/directory"
+    }
+}
+```
+
+But it **didn't** work out.
```

Comment:
No need to document this then, it is fine if we just say that Safari doesn't support an defining download dirs through WebDriver capabilities